### PR TITLE
Restore `swap-branches-on-compare` feature

### LINKED ISSUE #4223: `swap-branches-on-compare` not working?
The arrow isn't clickable

gif

URL: https://github.com/sindresorhus/refined-github/compare/main

Follows: https://github.com/sindresorhus/refined-github/issues/1217

### PR DESCRIPTION
**Pull request by ceelo777**
Closes #4223 
https://github.com/sindresorhus/refined-github/compare/main...Slash/slash
image
image

### CODE REVIEW

File: source/features/swap-branches-on-compare.tsx [hunk 1]
```diff
@@ -12,13 +12,14 @@ function init(): void {
...
-
+	
 	// Compares against the "base" branch if the URL only has one reference
 	if (references.length === 1) {
 		references.unshift(select('.branch span')!.textContent!);
 	}
 
-	const icon = select('.octicon-arrow-left')!;
+	const icons = select.all('.octicon-arrow-left')!;
+	const icon = icons[icons.length - 1];
```
Comment (yakov116) [inline source/features/swap-branches-on-compare.tsx:22]:
```suggestion
	const icon = select.all('.range-editor .octicon-arrow-left')!;
```

Does this work?

File: source/features/swap-branches-on-compare.tsx [hunk 2]
```diff
@@ -12,13 +12,14 @@ function init(): void {
 		.replace('compare/', '')
 		.split('...')
 		.reverse();
-
+
```
Comment (yakov116) [inline source/features/swap-branches-on-compare.tsx:15]:
```suggestion
```
Added by mistake

**Comment by yakov116** [comment]
@ceelo777 Very good job!

Can you fill in the issue template?

File: source/features/swap-branches-on-compare.tsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ function init(): void {
 		references.unshift(select('.branch span')!.textContent!);
 	}
 
-	const icon = select('.octicon-arrow-left')!;
+	const icon = select('.range-editor .octicon-arrow-left')!;
```
Comment (yakov116) [inline source/features/swap-branches-on-compare.tsx:21]:
```suggestion
	const icon = select('.range-editor .octicon-arrow-left')!;
```

You left an extra tab

**Comment by fregante** [comment]
What other arrow is on the page? Do they have another icon that shows up in small viewports? Do we have to also include that?

**Comment by yakov116** [comment]
> What other arrow is on the page? Do they have another icon that shows up in small viewports? Do we have to also include that?

Repo watch setting

**Comment by yakov116** [comment]
Thanks @ceelo777

**Pull request merged**